nextflow.config

```diff
@@ -264,57 +270,93 @@ env {
-set -C # No clobber - prevent output redirection from overwriting files.
-"""
+process.shell = [
+    "bash",
+    "-C",         // No clobber - prevent output redirection from overwriting files.
+    "-e",         // Exit if a tool returns a non-zero status/exit code
+    "-u",         // Treat unset variables and parameters as an error
+    "-o",         // Returns the status of the last command to exit..
+    "pipefail"    //   ..with a non-zero status or zero if all successfully execute
+]
 
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/riboseq'
-    author          = """Jonathan Manning"""
+    author          = """Jonathan Manning""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'Jonathan Manning',
+            affiliation: 'Seqera',
+            email: '',
+            github: '@pinin4fjords',
+            contribution: ['author'],
+            orcid: '0000-0002-3483-8456'
+        ],
+        [
+            name: 'Felix Krueger',
+            affiliation: 'Altos Labs',
+            email: '',
+            github: '@FelixKrueger',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Jack Tierney',
+            affiliation: 'CRT Genomic Data Science, University College Cork',
+            email: '',
+            github: '@JackCurragh',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Maxime U Garcia',
+            affiliation: 'Seqera',
+            email: '',
+            github: '@maxulysse',
+            contribution: ['contributor'],
+            orcid: ''
```

Comment:
```suggestion
            email: 'maxime.garcia@seqera.io',
            github: '@maxulysse',
            contribution: ['contributor'],
            orcid: '0000-0003-2827-9261'
```

Reply:
You might want to ask the other contributors for their details

Reply:
They weren't in there before, this is just a format change for me, they can be added over time if folks want.